applications/FluidDynamicsApplication/tests/two_fluid_mass_conservation_inlet_test.py

```diff
@@ -0,0 +1,152 @@
+from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+import KratosMultiphysics
+import KratosMultiphysics.FluidDynamicsApplication
+from KratosMultiphysics.FluidDynamicsApplication.fluid_dynamics_analysis import FluidDynamicsAnalysis
+
+import math
+import time
+import os
+import sys
+
+import KratosMultiphysics.KratosUnittest as UnitTest
+
+# Class to navigate through the folders
+class WorkFolderScope:
+    def __init__(self, work_folder):
+        self.currentPath = os.getcwd()
+        self.scope = os.path.abspath(os.path.join(os.path.dirname(os.path.realpath(__file__)),work_folder))
+
+    def __enter__(self):
+        os.chdir(self.scope)
+
+# Class derived from the UnitTest (KratosMultiphysics.KratosUnittest) class
+class TwoFluidMassConservationInletTest(UnitTest.TestCase):
+
+    def __init__(self):
+        self.work_folder = "TwoFluidMassInletTest"
+        self.check_absolute_tolerance = 1.0e-7
+        self.check_relative_tolerance = 1.0e-5
+        self.print_output =True
+        self.print_reference_values =True
```

Comment:
```suggestion
        self.print_output =False
        self.print_reference_values =False
```
Otherwise you'll be always printing the output and the reference values (meaning that nothing is checked).